sdk/servicebus/azservicebus/receiver.go

```diff
@@ -0,0 +1,24 @@
+package azservicebus
+
+type ReceiveMode int
+
+const (
+	ReceiveModePeekLock         ReceiveMode = 0
+	ReceiveModeReceiveAndDelete ReceiveMode = 1
+)
```

Comment:
Do these need to have the "ReceiveMode" prefix? It makes the consts even longer and the second almost has stuttering

Reply:
I removed the 'ReceiveMode' part as you suggested. 

I'm not entirely sure if they're needed or not - it makes discovery slightly easier if you're just going off the package but perhaps it's not needed since the 'type' is there and they're grouped together in the source.

@jhendrixMSFT - is there a standard around this one?